eng/pipelines/runtime-linker-tests.yml

```diff
@@ -79,3 +79,21 @@ jobs:
       extraStepsTemplate: /eng/pipelines/libraries/execute-trimming-tests-steps.yml
       extraStepsParameters:
           extraTestArgs: '/p:WasmBuildNative=false'
+
+#
+# Build Release config vertical for Browser-wasm_win
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Browser_wasm_win
+    jobParameters:
+      testGroup: innerloop
+      timeoutInMinutes: 120
+      nameSuffix: Windows_Runtime_Release
+      buildArgs: -s mono+libs -c $(_BuildConfig) -p:WasmBuildNative=false
```

Comment:
here .. `/p:WasmBuildNative=false`

Reply:
Can one of these be removed or are they both needed?

Reply:
Good question! The first one is for the build itself. And the second one is for the `extraStep`, which runs the tests.

`extraTestArgs` is being used in only one [place](https://github.com/dotnet/runtime/blob/main/eng/pipelines/libraries/execute-trimming-tests-steps.yml#L7), to build the the command line for running tests. So, the first one wouldn't imply the second.

> Sorry for all the questions haha.

Feel free to keep asking ;)

Reply:
Ok thank you. That makes a lot of sense